util/metrics/src/service_metrics.rs

```diff
@@ -122,26 +128,42 @@ impl ServiceMetrics {
         )
     }
 
+    /// Gets the method name from a gRPC RpcContext.
+    pub fn get_method_name(ctx: &RpcContext) -> GrpcMethodName {
+        let mut method_name = "unknown_method".to_string();
```

Comment:
Pretty insignificant but a match statement would be shorter

Reply:
Done.